src/main/java/com/mvelyka/library/Library.java

```diff
@@ -0,0 +1,26 @@
+package com.mvelyka.library;
+
+public class Library {
+    Book[] books;
+
+    public Library(Book[] books) {
+        this.books = books;
+    }
+    public void borrowBook(String bookName) {
+        for (int i = 0; i < books.length; i++) {
```

Comment:
borrowBook and returnBook is almost the same. We could refactor this into base private method.